Platformio/hardware/windows_linux/util.cpp

```diff
@@ -0,0 +1,11 @@
+unsigned long reverseBits(unsigned long n) {
```

Comment:
for some wild reason the remote codes in the flirc sdk are backwards from the arudino library